src/utils/components/DiskModal/components/StorageProfileSettings/VolumeMode.tsx

```diff
@@ -1,58 +1,69 @@
-import React, { FC, useEffect, useMemo } from 'react';
-import { Controller, useFormContext } from 'react-hook-form';
+import React, { FC, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { uniq } from 'lodash';
```

Comment:
Since when have we entered loadsh in the project? @metalice @upalatucci @pcbailey 
I thought we wanted to avoid this package as it's big and not needed
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore, please find the the replacement here and and it to same location as isEmpty is found

Reply:
It seems we need to discuss the strategy:) I've used it because it's already there. Note that there are modern alternatives like [radash](https://radash-docs.vercel.app/docs/getting-started)

Reply:
@avivtur the approach in https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_uniq cannot be used as we need to keep the initial order (most recommended come first).

Reply:
how do u know the recommendation?